### PR TITLE
Repl ccloud to ccloud

### PR DESCRIPTION
**Pull request by MosheBlumbergX**
Workflow for confluent cloud to confluent cloud replication with control center in the background to look at the Monitoring Extension metrics.

### CODE REVIEW

**Comment by MosheBlumbergX** [comment]
@rohit2b  , can you check this one out please?

**Review by rohit2b — CHANGES_REQUESTED** [review]
Added one suggestion to start the README. Apart from that, this looks good!

File: hybrid/replicator-cloud2cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# Replicator
+
```
[the remark this answers is not in the document]
Comment (MosheBlumbergX) [reply hybrid/replicator-cloud2cloud/README.md:2]: thanks @rohit2b , now added.

**Review by rohit2b — APPROVED** [review]
LGTM!

**Pull request merged**